CondTools/L1Trigger/interface/L1ConfigOnlineProdBase.h

```diff
@@ -161,7 +161,7 @@ L1ConfigOnlineProdBase<TRcd, TData>::produce( const TRcd& iRecord )
 	 if( !payloadToken.empty() )
 	   {
 	     m_dbSession.transaction().start() ; 
-	     pData = m_dbSession.fetchPayload<TData>( payloadToken ) ;
+	     pData = std::make_unique<TData>(*(m_dbSession.fetchPayload<TData>( payloadToken ).get())) ;
```

Comment:
@Dr15Jones fetchPayload returns a shared_ptr. Here I make a copy of the object and create a unique_ptr for return.

Reply:
In this case, I think it would be better to just keep return a `std::shared_ptr`.